doc/conf_ref.rst

```diff
@@ -75,7 +75,8 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     When upgrading a package, always try to install its highest version
     available, even only to find out some of its deps are not
     satisfiable. Enable this if you want to experience broken dependencies in
-    the repositories firsthand. The default is False.
+    the repositories firsthand. The default is ``True``. The default can be overridden by
+    :ref:`DNF configuration file <conf_ref-label>` that could differ for particular distribution.
```

Comment:
To me the sentence:
> The default can be overridden by :ref:`DNF configuration file <conf_ref-label>`

points to the _DNF Configuration Reference_ section, which this paragraph is part of, therefore it references itself, which is confusing.

If you want to reference the first paragraph of _Description_ talking about the main config file and the .repo configs, IMO you need to separate that paragraph and reference directly to it, as well as phrase this in a bit more concrete way to point out what you mean by the reference. I also think this is really redundant. The user is reading the configuration reference documentation, they should be able to scroll up easily to find out where the files are. The text isn't long either.

If you put this here, you might as well put it in the description of every option listed here, which doesn't really make sense.